### ISSUE TITLE
Bug : UpdateAir Moist Air faillure

### ISSUE THREAD

**Issue by kurupt44** [MadsKirkFoged/SharpFluids#28]
Hello,

Moist Air Update with HumidityRatio and RelativeHumidity doesn't work. 
The exception message is:
Value was either too small or too large for a Decimal.

I try with this :
Air.UpdateAir(Pressure.FromSI(101325), HumidityRatio: ParamIn2Value, RelativeHumidity: ParamIn1Value);
ParamIn2Value is double, 0.011
ParamIn1Value is double, 0.8 

Thank you

**Comment by MadsKirkFoged** [comment]
The problem was that Coolprop is just returning Inf when trying to do the lookup with your input vaules.
I have changed the code so the values is displaying Inf.

Are you sure the combination of values are valid inputs?

**Comment by kurupt44** [comment]
Yes the combination of values is valid, I try with many others and the result is the same.
Now returned value of all updated parameters are inf but this is not the correct result

**Comment by MadsKirkFoged** [comment]
I have checked everything and it is sending the correct values to CoolProp.
Maybe Coolprop dont support that combination of inputs..
I think if you want an answer for this question you'll have to go ask on CoolProps github: https://github.com/CoolProp/CoolProp

**Comment by romarro** [comment]
it shouldn't be possible to set both the RelativeHumidity and HumidityRatio. There isn't a single set of pressure and temperature values that can satisfy these two values. 
If you set pressure, temperature and RelativeHumidity this will result in a HumidityRatio, or the other way around.  These two quantities relates to the same physical quantity, the mass of the water vapours in air.
